src/test/java/resourceTests/PersonsResourcesTest.java

```diff
@@ -77,6 +78,7 @@ public void testGetNote()throws ParseException{
         verify(mockNotesDao, times(1)).getByPersonIdAndDate(1L,date);
     }
 
+    @Ignore
```

Comment:
Because right now we can always add a note, and when we later add authentication, tests for failing to add a note will be needed.